README.md

```diff
@@ -269,7 +270,15 @@ Here is a sample configuration:
   }
 }
 ```
+### FhirVersion Specify
+| fhirVersion | Desciption |
+| ----- | ----- |
+|Stu3|Specify STU 3 version for the configuration file|
+|R4|Specify R4 version for the configuration file|
+|Empty or Null| The configuration file will be specified the same version with the running executable file.
 
+> **[!NOTE]**
+> When  _fhirVersion_ section is missing or empty, the program will not throw an exception but give a warning information.

```

Comment:
Seems this section is not necessary as we have explained in line 278

Reply:
But we haven't explain the warning information in the table.

Reply:
For customers, this warning is not worth emphasizing here.

Reply:
Is the warning always printed (both with -v and without -v)?

Reply:
always.